src/character.cpp

```diff
@@ -2244,53 +2254,58 @@ hp_part Character::body_window( const std::string &menu_header,
-                                                 texitify_healing_power( new_b_power ) ), c_light_green ) << "\n";
+            desc << plus << arrow << tag_colored_string( string_format( _( "Expected bandage quality: %s" ),
+                    texitify_healing_power( new_b_power ) ), c_light_green ) << "\n";
         }
-
-        // DISINFECTANT / BITTEN / INFECTED block
-        if( disinfected ) {
-            desc << string_format( _( "Disinfected [%s]" ),
-                                   texitify_healing_power( d_power ) ) << "\n";
-            if( new_d_power > d_power ) {
-                desc << arrow << tag_colored_string( string_format( _( "Expected quality improvement: %s" ),
-                                                     texitify_healing_power( new_d_power ) ), c_light_green ) << "\n";
-            } else if( new_d_power > 0 ) {
-                desc << arrow << tag_colored_string( _( "You don't expect any improvement from using this item." ),
-                                                     c_yellow ) << "\n";
-            }
-        } else if( bitten ) {
+        // BITTEN block
+        if( bitten ) {
             desc << tag_colored_string( string_format( "%s: ", get_effect( effect_bite,
                                         e.bp ).get_speed_name() ), c_red );
-            desc << tag_colored_string( string_format( _( "It has a deep wound that needs cleaning." ) ),
+            desc << tag_colored_string( string_format( _( "It has a deep bite wound that needs cleaning." ) ),
                                         c_red ) << "\n";
             if( bite > 0 ) {
                 desc << arrow << tag_colored_string( string_format( _( "Chance to clean and disinfect: %d %%" ),
                                                      int( bite * 100 ) ), c_light_green ) << "\n";
-                if( new_d_power > 0 ) {
-                    desc << arrow << tag_colored_string( string_format( _( "Expected disinfection quality: %s" ),
-                                                         texitify_healing_power( new_d_power ) ), c_light_green ) << "\n";
-                }
+            } else {
+                desc << arrow << tag_colored_string( _( "This item will not help in cleaning this wound." ),
+                                                     c_yellow ) << "\n";
             }
-        } else if( infected ) {
+        }
+        // INFECTED block
+        if( infected ) {
             desc << tag_colored_string( string_format( "%s: ", get_effect( effect_infected,
                                         e.bp ).get_speed_name() ), c_red );
-            desc << tag_colored_string( string_format( _( "It has a deep wound that looks infected." ) ),
-                                        c_red ) << ">\n";
+            desc << tag_colored_string( string_format(
+                                            _( "It has a deep wound that looks infected. You might require antibiotics." ) ),
```

Comment:
I'm not native speaker, but "You might require" sounds strange to me. "It might require" perhaps?

Reply:
"Antibiotics might be required." would work well for both player and NPC.